evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -320,6 +321,26 @@ impl<'a, B: Backend> CheatcodeStackExecutor<'a, B> {
 
         Capture::Exit((ExitReason::Succeed(ExitSucceed::Stopped), res))
     }
+    fn console_log(&mut self, input: Vec<u8>) -> Capture<(ExitReason, Vec<u8>), Infallible> {
+        let decoded = match ConsoleLogCalls::decode(&input) {
+            Ok(inner) => inner,
+            Err(err) => return evm_error(&err.to_string()),
+        };
+        match decoded {
+            ConsoleLogCalls::Log(inner) => {
+                println!("Decoded Data: {:?}", inner);
```

Comment:
```suggestion
                println!("{}", inner);
```

Same for the ones below